### DISCUSSION TITLE
Images rendering double

### DISCUSSION THREAD

**Discussion by ZeldaFan0225** [Bodmer/TFT_eSPI#2045] [Q&A - General]
![image](https://user-images.githubusercontent.com/78901316/192334738-63255f17-b0ff-4f41-968f-1f9067af1ede.jpg)

Hey there I am trying to render 2 Images of an arrow pointing left and right. 

If I render the arrow left image twice it only shows it twice and vice versa and if I render both left and right for some reason it is rendering one button twice even though not specified in the code...

i use 

tft.pushImage(X, Y, imageWidth, imageHeight, ImageImport)
tft.pushImage(X, Y, imageWidth, imageHeight, ImageImport)

exactly as shown in the example in the repo...

**Comment by Bodmer** [comment]
I suspect the problem is elsewhere in the code or an incorrectly sized image in the wrong format.  A complete working sketch would be needed to identify the route cause.
